docs/resources/meeting_conferences.md

```diff
@@ -0,0 +1,527 @@
+
+* `audience_join_uri` - The audience meeting link address.
+
+* `subconferences` - The list of periodic sub-conferences.
+  The [object](#conference_subconferences) structure is documented below.
+
+<a name="conference_subconferences"></a>
+The `subconferences` block supports:
+
+* `id` - The sub-conference ID.
+
+* `media_types` - The media type list.
+
+* `start_time` - The sub-conference start time.
+
+* `end_time` - The sub-conference end time.
+
+* `is_auto_record` - Whether to automatically start recording.
+
+* `record_auth_type` - The recording authentication method.
+
+* `subconfiguration` - The other configuration information of periodic subconferences.
+  The [object](#conference_subconfiguration) structure is documented below.
+
+<a name="conference_subconfiguration"></a>
+The `subconfiguration` block supports:
+
+* `callin_restriction` - The range to allow incoming calls.
+
+* `audience_callin_restriction` - The range that the webinar audience is allowed to call in.
+  The valid values are as follows:
+  + **0**: All users.
+  + **2**: Users within the enterprise.
+
+* `allow_guest_start` - Whether to allow guests to start conferences (only valid for random ID conferences).
+
+* `waiting_room_enabled` - Whether to open the waiting room (only valid for RTC enterprises).
+
+* `show_audience_policy` - The webinar Audience Display Strategy.
+  The [object](#conference_show_audience_policy) structure is documented below.
+
+<a name="conference_show_audience_policy"></a>
+The `show_audience_policy` block supports:
+
+* `mode` - Audience display strategy: The server is used to calculate the number of audiences and send it to the client
+  to control the audience display.
+  + **0**: Do not display.
+  + **1**: Multiply display the number of participants, based on the real-time number of participants or the cumulative
+  number of participants, the multiplication setting can be performed.
+
+* `multiple` - Specifies the multiplier. The valid values is range from `0` to `10`, it can be set to 1 decimal place.
+
+* `base_audience_count` - Specifies the basic number of people, the valid values is range from `0` to `10,000`.
+
+-> After `base_audience_count` and `multiple` are both setting, the number of people displayed is: **NX+Y**.
+  When NX calculates a non-integer, it will be rounded down.
+
+## Import
+
+Conferences (only scheduled conference and progressing conference) can be imported using their `id` and authorization
+parameters, separated by slashes, e.g.
+
+Import a conference and authenticated by account.
+
+```
+$ terraform import huaweicloud_meeting_conference.test &ltconference_uuid&gt/&ltconference_id&gt/&ltaccount_name&gt/&ltaccount_password&gt
+```
+
+Import a conference and authenticated by `APP ID`/`APP Key`.
+
+```
+$ terraform import huaweicloud_meeting_conference.test &ltconference_uuid&gt/&ltconference_id&gt/&ltapp_id&gt/&ltapp_key&gt/&ltcorp_id&gt/&ltuser_id&gt
```

Comment:
addd a not here as the corp_id and user_id can be empty

Reply:
done.